nomos-blend/network/src/core/with_core/behaviour/tests/utils.rs

```diff
@@ -112,7 +149,19 @@ impl BehaviourBuilder {
                 .minimum_network_size
                 .unwrap_or_else(|| 1usize.try_into().unwrap()),
             old_session: None,
-            poq_verifier: AlwaysTrueVerifier,
+            poq_verifier: ProofsVerifier::new(PoQVerificationInputsMinusSigningKey {
```

Comment:
Do you think we should expose a method on the builder to specify these inputs, and use the default if no value is passed? Else this assumes that all verifiers will only work with `ZkHash::ZERO` values, but that is not necessarily true. So we should expose a function to specify these values.